### PR TITLE
Create newsletter-2018-august.rst

### PR DESCRIPTION
**Pull request by kelly-o**

### CODE REVIEW

File: docs/blog/newsletter-2018-august.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Having this conversation with a bunch of other documentarians meant there was no shortage of support and commiseration to be found. When it comes to showing other people the error of their ways, one practical suggestion was to lean on whatever analytics or reporting you have – and if you don't have any, start collecting some. Another, perhaps less serious suggestion was to just stop writing the docs for a month and see how things go then. ;)
+
+
+*The* `#career-advice <https://writethedocs.slack.com/messages/career-advice>`_ *channel is always full of sympathetic ears and practical advice for whatever professional frustrations you may encounter.*
+
+---------------------------------------------------------------------------
+In the time of web-based applications, what is a page and what is a screen?
+---------------------------------------------------------------------------
+
+An classic question of word choice arose this month: What's the difference between a "page", a "screen" – and which term should be used when?
```
Comment (clairealyssa) [inline docs/blog/newsletter-2018-august.rst]: Should be "A classic question" right? 😄

File: docs/blog/newsletter-2018-august.rst [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+* Others prefer Hugo to Sphinx because it seems to be more widely adopted, offers a wider range of themes from which to choose, and is more easily extensible. Extending Sphinx or developing a custom theme for Sphinx deliverables relies on in-house Python knowledge.
+* Gatsby offers broader functionality because it is built on top of React/Node and GraphQL. Unlike other SSGs, including Sphinx, it can build content drawn from multiple sources. Setting up a Gatsby pipeline can require more coding than with other SSGs, but can provide more powerful functionality if that's what you need.
+
+This month's conversation was by no means exhaustive or conclusive. There are so many different approaches to building an SSG publishing workflow, and a lot of enthusiasm and knowledge around the pros and cons of each. You can definitely expect to see more where this conversations came from.
+
+*Experimenting with different SSGs yourself?: Pop over to* `#static-site-generator <https://writethedocs.slack.com/messages/static-site-generator>`_ *or see if there's a dedicated channel for the particular SSG you've got your eye on.*
```
Comment (clairealyssa) [inline docs/blog/newsletter-2018-august.rst]: The ?: combo looks weird.

File: docs/blog/newsletter-2018-august.rst [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+   :tags: newsletter
+
+
+#######################################
+Write the Docs Newsletter – August 2018
+#######################################
+
+Hi everybody!
+
+Kelly O. here with your late summer issue of the Write the Docs newsletter! After a nice leisurely few weeks, mid-summer, things are definitely getting back into the swing of things in here docsland. Team Newsletter's back on the case. The Prague conference is sold out and will be upon us in less than 3 weeks. And happening as we speak is the first ever Cincinnati Write the Docs + Open Help event!
```
Comment (hillaryfraley) [inline docs/blog/newsletter-2018-august.rst]: we are definitely (instead of things are definitely)?

Comment (hillaryfraley) [reply docs/blog/newsletter-2018-august.rst]: also, "first-ever" (add hyphen)

File: docs/blog/newsletter-2018-august.rst [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+If you'd be interested in learning more, drop me an email at newsletter@writethedocs.org or ping me on Slack (@kobrien042) and I'll be happy to tell you more!
+
+In the meantime, let's tune into some highlights from the hiatus!
+
+-----------------------------------------
+Common misconceptions about documentation
+-----------------------------------------
+
+Inspired by `Emily January Petersen's guest post <http://idratherbewriting.com/2018/07/18/stereotypes-about-tech-writers-in-workplace/>`_ on `I'd Rather Be Writing <http://idratherbewriting.com>`_, documentarians discussed some common misconceptions about our profession that they've personally experienced. As one person pointed out, **we** know that it takes more than being able to use a word processor or a text editor to be a documentarian... but what about everyone else?
```
Comment (hillaryfraley) [inline docs/blog/newsletter-2018-august.rst]: Remove the space after the ellipsis

File: docs/blog/newsletter-2018-august.rst [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+Here are some of the unfortunately common misconceptions mentioned in Slack this month:
+
+- Anyone can write good docs.
+- No one reads the docs.
+- Documentation is fast and easy to produce.
+- Documentation doesn't have any discernible business value.
+- Tech writers are just there to make docs pretty.
+- It's not a role that earns much respect or recognition.
+
+Having this conversation with a bunch of other documentarians meant there was no shortage of support and commiseration to be found. When it comes to showing other people the error of their ways, one practical suggestion was to lean on whatever analytics or reporting you have – and if you don't have any, start collecting some. Another, perhaps less serious suggestion was to just stop writing the docs for a month and see how things go then. ;)
```
Comment (hillaryfraley) [inline docs/blog/newsletter-2018-august.rst]: Maybe remove the comma after "Another"?

File: docs/blog/newsletter-2018-august.rst (hunk 1 above)
Comment (hillaryfraley) [inline docs/blog/newsletter-2018-august.rst]: What's the difference between a page and a screen - and which term should be used when?

**Review by clairealyssa — CHANGES_REQUESTED** [review]
Thank you so much Kelly.

File: docs/blog/newsletter-2018-august.rst (hunk 1 above)
Comment (kelly-o) [reply docs/blog/newsletter-2018-august.rst]: It most certainly should... ;)

**Pull request merged**